packages/docker_otel_input/manifest.yml

```
@@ -0,0 +1,74 @@
+format_version: 3.5.0
+name: docker_otel_input
+title: "Docker (OpenTelemetry)"
+version: 0.2.0
+description: "Collect Docker container metrics using OpenTelemetry Collector"
+type: input
+categories:
+  - containers
+  - monitoring
+  - observability
+  - opentelemetry
+conditions:
+  kibana:
+    version: "^9.2.0"
+  elastic:
+    subscription: "basic"
+icons:
+  - src: /img/docker_logo.svg
+    title: Sample logo
+    size: 32x32
+    type: image/svg+xml
+policy_templates:
+  - name: docker_stats
```

This should be the name of the receiver, as this is used by Fleet as default dataset. In this case it will likely be `dockerstatsreceiver`.

We will prepare automatic validation for this in https://github.com/elastic/elastic-package/issues/2974
```suggestion
  - name: dockerstatsreceiver
```